tests/ops/op_math/test_exp.py

```diff
@@ -796,6 +794,47 @@ def circ(phi):
         grad = jax.grad(circ)(phi)
         assert qml.math.allclose(grad, -jnp.sin(phi))
 
+    @pytest.mark.catalyst
+    @pytest.mark.external
+    def test_catalyst_qnode(self):
+        """Test with Catalyst interface"""
+
+        pytest.importorskip("catalyst")
+
+        phi = 0.345
+
+        @qml.qjit
+        @qml.qnode(qml.device("lightning.qubit", wires=1))
+        def func(params):
+            qml.exp(qml.X(0), -0.5j * params)
+            return qml.expval(qml.Z(0))
+
+        res = func(phi)
+        assert qml.math.allclose(res, np.cos(phi))
+        grad = qml.grad(func)(phi)
+        assert qml.math.allclose(grad, -np.sin(phi))
+
+    @pytest.mark.jax
+    @pytest.mark.external
```

Comment:
Why the external marker? I only see jax being used.

Reply:
Removed